aicsimageio/writers/ome_tiff_writer.py

```diff
@@ -447,7 +470,9 @@ def dim_or_1(dim: str) -> int:
         if isinstance(channel_names, list) and len(channel_names) != channel_count:
             raise ValueError(f"Wrong number of channel names {len(channel_names)}")
         if isinstance(channel_colors, list) and len(channel_colors) != channel_count:
-            raise ValueError(f"Wrong number of channel colors {len(channel_colors)}")
+            raise ValueError(
+                f"Wrong number of channel colors {len(channel_colors)} {channel_colors}"
```

Comment:
Should we include `channel_count` in this exception so users know what the expected number of colors should be?

Reply:
Eh nah. We have other checks for that. We have a check somewhere that asserts that all metadata parameters are equal and such.

Reply:
Oh wait. I'm wrong. Yea I will add that.